src/economy/demographics.cpp

```diff
@@ -63,7 +63,7 @@ uint32_t size(sys::state const& state) {
 				 uint32_t(2) * state.world.pop_type_size() + state.world.culture_size() + state.world.religion_size();
 }
 uint32_t common_size(sys::state const& state) {
-	return count_special_keys + uint32_t(2) * state.world.pop_type_size();
+	return count_special_keys;
```

Comment:
the pop size bit was intentional, because I want to force the per-pop-type population numbers and the per-pop-type employment numbers to be updated daily (those numbers are used by the economy update in a couple places, and so I considered them more crucial to keep up to date correctly)

Reply:
I see now, reverted.